### DISCUSSION TITLE
the IoC container guide

### DISCUSSION THREAD

**Discussion by artem-a** [adonisjs/core#2578] [General]
Hi there. Where can I find the IoC container guide? I've read a new documentation and seen this note "We recommend reading the IoC container guide to understand the concept of namespaces.". I've searched it in the new docs but I couldn't find it there. Is it in the docs or it'll be soon there?

Thanks.

**Comment by RomainLanz** [comment]
Hey @artem-a! 👋 

Some advance concepts are still not documented, it takes time to find a way to explain them well.
It will be documented later on. 👍
